examples/app_media_source/port/ameba_pro2/ameba_pro2_media_port.c

```diff
@@ -250,7 +250,7 @@ static int HandleModuleFrameHook( void * p,
             frame.freeData = 1;
             frame.timestampUs = NetworkingUtils_GetCurrentTimeUs( &pInputItem->timestamp );
 
-            if( pInputItem->type == AV_CODEC_ID_H264 )
+            if( pInputItem->type == AV_CODEC_ID_H264 || pInputItem->type == AV_CODEC_ID_H265 )
```

Comment:
```suggestion
            if( ( pInputItem->type == AV_CODEC_ID_H264 ) || ( pInputItem->type == AV_CODEC_ID_H265 ) )
```
Nit to align coding convension.